### PR TITLE
builtin cd overrides user aliases and functions

### PR DESCRIPTION
**Pull request by MenkeTechnologies**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

builtin cd to override user alias and functions in system OMZ functions. Important when capturing output and writing to zcompdump

~/oh-my-zsh/oh-my-zsh.sh: 66 #omz revision: $(builtin cd -q "$ZSH"; git rev-parse HEAD 2>/dev/null)## Other comments:

## Other comments:

tested

### CODE REVIEW

File: tools/upgrade.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ else
   RESET=""
 fi
 
-cd "$ZSH"
+builtin cd "$ZSH"
```
Comment (mcornella) [inline tools/upgrade.sh:20]: `upgrade.sh` is ran with `sh`, `builtin` doesn't exist here.

Comment (MenkeTechnologies) [reply tools/upgrade.sh:20]: ok, i will remove this one.

Comment (MenkeTechnologies) [reply tools/upgrade.sh]: done

**Comment by mcornella** [comment]
Thanks!

**Pull request merged**